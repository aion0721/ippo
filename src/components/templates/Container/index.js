import React from 'react';
import styled from 'styled-components';
import NavigationBar from 'src/components/molecules/NavigationBar';

const StyledContainer = styled.div`
  padding: 10px 15px;
`;

const Container = ({ children }) => (
  <>
    <NavigationBar appName="IPPO" />
    <StyledContainer>{children}</StyledContainer>
  </>
);

export default Container;
