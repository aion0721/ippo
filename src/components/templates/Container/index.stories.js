import React from 'react';
import { storiesOf } from '@storybook/react';
import { text } from '@storybook/addon-knobs';
import Container from '.';

const stories = storiesOf('templates/Container', module);

const props = {
  children: text('children', 'コンテンツ'),
};

stories.add('通常パターン', () => <Container {...props} />);
