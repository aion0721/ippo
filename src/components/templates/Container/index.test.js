import React from 'react';
import { snapshot } from 'test/helpers';
import Container from '.';

const props = {
  children: 'test',
};

snapshot('Container/nomal', <Container {...props} />);
