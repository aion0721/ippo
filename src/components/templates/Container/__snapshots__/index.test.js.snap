// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: Container/nomal 1`] = `
.c0 {
  padding: 10px 15px;
}

<Container>
  <NavigationBar
    appName="IPPO"
  >
    <Blueprint3.Navbar>
      <div
        className="bp3-navbar"
      >
        <Blueprint3.NavbarGroup
          align="left"
        >
          <div
            className="bp3-navbar-group bp3-align-left"
          >
            <Blueprint3.NavbarHeading>
              <div
                className="bp3-navbar-heading"
              >
                IPPO
              </div>
            </Blueprint3.NavbarHeading>
          </div>
        </Blueprint3.NavbarGroup>
      </div>
    </Blueprint3.Navbar>
  </NavigationBar>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  padding: 10px 15px;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="c0"
      >
        test
      </div>
    </StyledComponent>
  </styled.div>
</Container>
`;
